### PR TITLE
Script for rendering test files

### PR DESCRIPTION
**Pull request by mrsabath**

### CODE REVIEW

File: utils/renderTestFiles.sh [hunk 1]
```diff
@@ -0,0 +1,102 @@
+#!/bin/bash -x
+
```
Comment (lumjjb) [inline utils/renderTestFiles.sh:2]: nit: make directories be based off the relative directory of script, using path relative to ` $(dirname "$0")`

File: utils/renderTestFiles.sh [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+    "tsi.secrets": "- tsi.secret/name: \"mysecret1\"\n  tsi.secret/role: \"tsi-role-rcni\"\n  tsi.secret/vault-path: \"secret/tsi-rcni\"\n  tsi.secret/local-path: \"mysecrets/myubuntu-mysecret1\"\n- tsi.secret/name: \"mysecret2.json\"\n  tsi.secret/role: \"tsi-role-rcni\"\n  tsi.secret/vault-path: \"secret/tsi-r\"\n  tsi.secret/local-path: \"mysecrets/myubuntu-mysecret2\"\n- tsi.secret/name: \"password\"\n  tsi.secret/role: \"tsi-role-rcni\"\n  tsi.secret/vault-path: \"secret/tsi-r\"\n  tsi.secret/local-path: \"mysecrets/myubuntu-passwords\"\n- tsi.secret/name: \"invalid\"\n  tsi.secret/role: \"tsi-role-rcni\"\n  tsi.secret/vault-path: \"secret/tsi-rcni\"\n  tsi.secret/local-path: \"mysecrets/myubuntu-invalid\"\n- tsi.secret/name: \"non-existing\"\n  tsi.secret/role: \"tsi-role-rcni\"\n  tsi.secret/vault-path: \"secret/nothing\"\n  tsi.secret/local-path: \"mysecrets/non-existing\"\n"
+}
+EOT
+sed 's/ubuntu@sha256:250cc6f3f3ffc5cdaa9d8f4946ac79821aafb4d3afc93928f0de9336eba21aa4/ubuntu@sha256:250cc6f3f3ffc5cdaa9d8f4946ac79821aafb4d3afc93928f0de9336eba21aa4,trustedseriviceidentity\/ti-jwt-sidecar:v1.5/g' ${TESTS}/FakeUpdateAnnotation.json > ${TESTS}/FakeUpdateAnnotation2.json
```
Comment (lumjjb) [inline utils/renderTestFiles.sh:35]: Does the versioning here i.e. `v1.5` have to change per iteration to work? do we need to use an env variable instead?

File: webhook.go [hunk 3]
```diff
@@ -723,9 +729,24 @@ func (whsvr *WebhookServer) serve(w http.ResponseWriter, r *http.Request) {
...
+			f, err := os.Create(filepath.Join(os.TempDir(), msg))
+			if err != nil {
+				panic(err)
+			}
+			_, err = f.WriteString(s)
+			if err != nil {
+				panic(err)
+			}
+			glog.Infof("Logged successfully to %s", f.Name())
+			defer f.Close()
```
Comment (lumjjb) [inline webhook.go:747]: nit: move defer up to after err checking for os.Create`

**Review by lumjjb — APPROVED** [review]
LGTM! Awesome, this definitely helps a lot for testing. Just a couple notes for minor nits!

File: utils/renderTestFiles.sh (hunk 2 above)
Comment (mrsabath) [reply utils/renderTestFiles.sh]: I removed that and made it completely dynamic

**Pull request merged**